### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
Remove the old glassfish server:
- Remove the proxy pass through
- fix one path
- remove not needed constructors
- remove micronaut api client
- remove old server files
- update the banner for the serve location

### CODE REVIEW

File: airbyte-server/src/main/resources/micronaut-banner.txt [hunk 1]
```diff
@@ -6,3 +6,6 @@
 /_/  |_/_/_/  /_.___/\__, /\__/\___/
                     /____/
         : airbyte-server :
+--------------------------------------------------
+ Will soon be available at http://localhost:8000/
+--------------------------------------------------
```
Comment (colesnodgrass) [inline airbyte-server/src/main/resources/micronaut-banner.txt:11]: Should we leave this message?

**Review by colesnodgrass — APPROVED** [review]

File: airbyte-server/src/main/resources/micronaut-banner.txt (hunk 1 above)
Comment (benmoriceau) [reply airbyte-server/src/main/resources/micronaut-banner.txt:11]: I have added it on purpose because the old server is available is not displayed anymore

**Review by gosusnp — APPROVED** [review]
clean up all the code

**Review by jdpgrailsdev — APPROVED** [review]
:shipit:

**Pull request merged**